src/store/models/taro.ts

```diff
@@ -66,7 +138,10 @@ const taroModel: TaroModel = {
           .filter(n => n.status === Status.Started)
           .map(async n => {
             try {
-              await actions.getAssets(n);
+              await (async () => {
+                actions.getAssets(n);
+                actions.getBalances(n);
```

Comment:
Call `actions.getAllInfo(n)` instead so that if we add any additional API calls there, this won't need to be updated as well.

Reply:
Done.